### ISSUE TITLE
Feature request - ZeroInflatedPoisson Distribution

### ISSUE THREAD

**Issue by ksachdeva** [tensorflow/probability#826]
Hi,

It would be great to have support for ZeroInflatedPoisson Distribution. Most of the other frameworks have support for this.

Regards
Kapil

**Comment by brianwa84** [comment]
Sounds like a good PR! Perhaps `ZeroInflated(underlying=Poisson(..), logits=..)`. That could also generalize nicely to `ZeroInflated(underlying=LogNormal(..), ...)`.

**Comment by ksachdeva** [comment]
@brianwa84 your api suggestion makes sense.

**Comment by StatMixedML** [comment]
any updates on this? Very relevant indeed!
